### DISCUSSION TITLE
Duplicate: initialized git in wrong directory

### DISCUSSION THREAD

**Discussion by tsekeres** [nss-evening-cohort-14/evening-client-side#25] [Help Me!] [answered]
Tad Sekeres

Have you searched the other help tickets? yes

Describe your issue: I have persistent  yellow X's in my workspace/foundations/exercises. I think they are latent X's left from the deleted test branches from our 30 Repo assignment. (wondering if it because I haven't pushed each again after deleting the branch)

List any error messages that appear (not an error message per se):
➜  exercises git:(main) ✗ git status 
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)

        modified:   product-cards (modified content)

Untracked files:
  (use "git add <file>..." to include in what will be committed)

        test07/
        test08/
        test09/
        test10/
        test11/
        test12/
        test13/
        test14/
        test15/
        test16/
        test17/
        test18/
        test19/

no changes added to commit (use "git add" and/or "git commit -a")
➜  exercises git:(main) ✗ cd test19

What have you googled?:
https://stackoverflow.com/questions/38512124/visual-studio-code-remove-branches-deleted-on-github-that-still-show-in-vs-cod
https://cloudskills.io/blog/git-azure-devops-2
https://code.visualstudio.com/Docs/editor/debugging

What have you tried?
I have tried to dig back into directories to find the unadded or uncommitted files.
I have git status'ed and can see the above messge. (not sure why the test titles are list with a / after them)

Copy/paste link TO THE FILE on your branch or in your repo:
Not sure how to link to something in my VS code.

**Comment by tsekeres** [comment]
Tried to Add again and:
➜  exercises git:(main) ✗ git add -A
warning: adding embedded git repository: test07
hint: You've added another git repository inside your current repository.
hint: Clones of the outer repository will not contain the contents of
hint: the embedded repository and will not know how to obtain it.
hint: If you meant to add a submodule, use:
hint: 
hint:   git submodule add <url> test07
hint: 
hint: If you added this path by mistake, you can remove it from the
hint: index with:
hint: 
hint:   git rm --cached test07
hint: 
hint: See "git help submodule" for more information.
warning: adding embedded git repository: test08
warning: adding embedded git repository: test09
warning: adding embedded git repository: test10
warning: adding embedded git repository: test11
warning: adding embedded git repository: test12
warning: adding embedded git repository: test13
warning: adding embedded git repository: test14
warning: adding embedded git repository: test15
warning: adding embedded git repository: test16
warning: adding embedded git repository: test17
warning: adding embedded git repository: test18
warning: adding embedded git repository: test19
➜  exercises git:(main) ✗ git rm --cached test07
error: the following file has staged content different from both the
file and the HEAD:
    test07
(use -f to force removal)

**Comment by drteresavasquez** [comment]
You have the parent directory initialized as a git repo and you DO NOT want that. See #6 on how to fix this issue.

**Reply by tsekeres** [reply]
I can't figure out what directory to be in when I run "rmdir .git".

**Reply by drteresavasquez** [reply]
You need to be in your exercises directory. 

The command is `rm -rf .git`

**Reply by tsekeres** [reply]
I may be wrong, but that web search page said that it is different for windows.

**Reply by drteresavasquez** [reply]
Oh! OK sorry. It is possible. Try what you have and let me know if that works for you.

**Reply by tsekeres** [reply]
Or am I running Linux on my Windows machine? Therefore making it a Linux fix?

**Reply by drteresavasquez** [reply]
You should be able to use the same commands that we all do, so if you are in the git bash shell or ubuntu, I think you can run what I specified above.

**Comment by drteresavasquez** [comment]
@tsekeres when you find a resolution, go ahead and let everyone know what worked for you by creating a new comment on this ticket.

**Comment by tsekeres** [comment, marked as answer]
I followed the process discussed in #6 to back up from having too many gits in one space. I used  rm -rf .git while in the directory affected.
